### ISSUE TITLE
missing output with consumeProcessOutput only when passed 'as Appendable' (the default)

### ISSUE THREAD

**Issue by jryan0** [kobo/groovyserv#52]
When running with groovyclient I get no output, with regular groovy I do:

```groovy
def p = 'dir'.execute()
p.consumeProcessOutput(System.out as Appendable, System.err as Appendable)
```
Also, if I switch to:
```groovy
def p = 'dir'.execute()
p.consumeProcessOutput(System.out as OutputStream, System.err as OutputStream)
```
It works.

Looks like it has something to do with the way newlines or reading the input stream is being handled differently by ProcessGroovyMethods.

**Comment by nobeans** [comment]
Fixed already, at v1.0.0 (1a665a8).

**Comment by jryan0** [comment]
Great! Thank you so much. Really appreciate it.
